client/my-sites/domains/domain-management/dns/style.scss

```diff
@@ -207,3 +207,38 @@ body.dns__body-white {
 		max-width: 600px;
 	}
 }
+
+a.dns__breadcrumb-button {
+	display: flex;
+	align-items: center;
+
+	&.add-record {
+		margin-right: 12px;
+	}
+
+	@include break-mobile {
+		&.add-record {
+			margin-right: 0;
+		}
+
+		svg {
+			margin-right: 8px;
+			fill: var( --color-neutral-70 );
+		}
+	}
+}
+
+.dns__breadcrumb-button {
+	.popover__menu-item[disabled] {
+		svg.gridicon {
+			fill: var( --color-neutral-20 );
+		}
+	}
```

Comment:
Having to define this feels like a regression 🤔 the icon was invisible because it was being colored `color: var( --color-neutral-0 )`. Or maybe we're defining something wrong and the icon ends up being colored wrongly?